docs/pipelines/index.md

```diff
@@ -134,3 +134,199 @@ else
 <!-- endSnippet -->
 
 Use `ExecuteOutcomeAsync(...)` in high-performance scenarios where you wish to avoid re-throwing exceptions. Keep in mind that Polly's resilience strategies also make use of the `Outcome` struct to prevent unnecessary exception throwing.
+
+## Diagrams
+
+### Variant one
```

Comment:
Let's explain roughly what these are in the headings, rather than just give it an arbitrary number.